Example/BonMot.xcodeproj/project.pbxproj

```diff
@@ -19,6 +19,7 @@
 		6003F5BA195388D20070C39A /* InfoPlist.strings in Resources */ = {isa = PBXBuildFile; fileRef = 6003F5B8195388D20070C39A /* InfoPlist.strings */; };
 		69EEA9487DE55634FFF1A253 /* Pods_BonMot_Tests.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = B6389DCE3A27CBCE74FD0558 /* Pods_BonMot_Tests.framework */; };
 		730361E31C9B5CDD00987809 /* BONHumanReadableStringTestCase.m in Sources */ = {isa = PBXBuildFile; fileRef = 730361E11C9B5CD300987809 /* BONHumanReadableStringTestCase.m */; };
+		9855962D1D1D7F19002E11CA /* BONLinkTestCase.m in Sources */ = {isa = PBXBuildFile; fileRef = 9855962C1D1D7F19002E11CA /* BONLinkTestCase.m */; };
```

Comment:
@plarson this must be in the `BonMot_Tests` target and _not_ in the `BonMot_Example` target.